### PR TITLE
[QA] Rest API acceptance tests

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers

@flopezag 
#### Description
- Refactor: GlanceSync CM Client
- Add: GlanceSync API Client
- Add: Acceptance tests for _API version_ resource.
- Refactor: qautils from external lib.
#### Testing

Locally.

### CODE REVIEW

File: tests/acceptance/features/api/api_version.feature [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  I want to see some information of the API that is running
+  So that I can check the version, owner, documentation location and other info parameters.
+
+  GlanceSync API operations:
+    GET /
+
+
+  Scenario: Request API information when user is authenticate.
+    Given the API running properly
+    And   the user is successfully authenticate
```
Comment (flopezag) [inline tests/acceptance/features/api/api_version.feature]: Take into account that the information do not need to be authenticated...

Comment (jframos) [inline tests/acceptance/features/api/api_version.feature]: Yes, I know.There are other scenarios that test the same flow but without authentication.

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
No merge, please. Jenkins is detecting some issue in this PR.

**Comment by jframos** [comment]
Ready to be merged. :+1:

**Pull request merged**